scripts/change_file_ext_test.py

```diff
@@ -0,0 +1,164 @@
+# coding: utf-8
+#
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for scripts/change_file_ext.py."""
+
+import os
+import shutil
+import tempfile
+
+from core.tests import test_utils
+from scripts import change_file_ext
+
+
+class ChangeExtensionsTest(test_utils.GenericTestBase):
+    """Test the change extension method."""
+
+    def setUp(self):
+        """Create dummy folders for testing and run the change extensions method
+        on the dummy files.
+        """
+        super(ChangeExtensionsTest, self).setUp()
+
+        self.rename_dir = tempfile.mkdtemp(suffix='dir_used_in_renaming')
+        self.non_rename_dir = tempfile.mkdtemp(
+            suffix='dir_not_used_in_renaming')
+
+        os.mkdir('%s/subdir' % self.rename_dir)
+        os.mkdir('%s/subdir' % self.non_rename_dir)
```

Comment:
Is the os.mkdir indisposable here? Can't `tempfile` perform this task? Because os.mkdir is still being used so the import of `tempfile` becomes ineffective. What do you think?

Reply:
When we create a temporary directory it goes to a default folder in our system. We can change the location of default folder where temporary directories and files are created but that will be common for all temporary items. 

Since in this case we need to create subdir in two of the folders, we need two different default folders for creation of temporary directories if we create subdir using `mkdtemp`, which is not possible. So, I used mkdir for creation of subdir.

Still it is effective to use tempfile since all the sub content resides in a temporary directory created  using tempfile which is safer than creating a directory in the codebase.

Does this make sense?

Reply:
Ah okay forgot about the two default path thing. Thanks for pointing that out! :)
LGTM then!

Reply:
No issues, thanks for the review :smile: